README.md

```diff
@@ -415,12 +415,81 @@ end
 
 You normally want to have your coverage analyzed across ALL of your test suites, right?
 
-Simplecov automatically caches coverage results in your (coverage_path)/.resultset.json. Those results will then
-be automatically merged when generating the result, so when coverage is set up properly for Cucumber and your
-unit / functional / integration tests, all of those test suites will be taken into account when building the
-coverage report.
+Simplecov automatically caches coverage results in your (coverage_path)/.resultset.json.
 
-There are two things to note here though:
+### Between test runs under the same environment
```

Comment:
I think _environmnet_ is a bit ambiguous as it might refer to dev/test/prod (yes it should usually be test) - maybe machine/execution environment or something else?

Reply:
Fixed by https://github.com/colszowka/simplecov/pull/780/commits/90d67d1c316273d69fef45db18ddf66fd6c425c0.